src/Aquifer.Data/Entities/ResourceContentVersionSimilarityScore.cs

```diff
@@ -0,0 +1,22 @@
+namespace Aquifer.Data.Entities;
+
+public class ResourceContentVersionSimilarityScore
+{
+    public int Id { get; set; }
```

Comment:
Wanted to use a composite primary key on `BaseVersionId` + `ComparedVersionId` but we are able to compare different content from different tables. So, a composite key will not work.